dataserver.js

```diff
@@ -0,0 +1,29 @@
+'use strict'; // eslint-disable-line strict
+require('babel-core/register');
+
+const arsenal = require('arsenal');
+const config = require('./lib/Config.js').default;
+const logger = require('./lib/utilities/logger').logger;
+
+if (config.backends.data === 'file') {
+    /**
+     * Configure the file paths for data if using the file
+     * backend. If no path provided, uses data at the root of
+     * the S3 project directory.
+     */
+    const dataPath = process.env.S3DATAPATH ?
```

Comment:
I moved it here, because it's not an S3 parameter anymore, but a "dmd" parameter. I think it makes sense to have it in this file because this is the startup file of the dmd. If you think it's better to keep everything in Config.js, I can revert back.